packages/dds/matrix/src/permutationvector.ts

```diff
@@ -360,10 +320,10 @@ export class PermutationVector extends Client {
 
 		// Notify the undo provider, if any is attached.
 		if (this.undo !== undefined && isLocal) {
-			this.undo.record(operation, ranges);
+			this.undo.record(deltaArgs);
```

Comment:
Do you know if there's any reason matrix has undo baked-in like this unlike string which is relying on the app to hook up the event to revert code?

Reply:
i don't know if there is a specific reason, likely just dev preference..